py/greetings.py

```diff
@@ -1,19 +1,36 @@
+import html
 import dominate
 from dominate.tags import *
 
-def _greetings():
-    """Returns the generated HTML greetings. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+def _greetings(week):
+    """This function generates a greetings section below the header using table elements and returns it as HTML. 
+    The greetings consists of two subheaders separated by a horizontal line. The first subheader prints the function argument,
+    which will most likely be "HAPPY WEEK # EVERYONE" and the second subheader prints, "Here are some things to get excited about!".
```

Comment:
This comment gets misleading if we are allowing the user to specify the entire string for the first subheader.

Reply:
Fixed